bok_choy/a11y/axs_ruleset.py

```diff
@@ -0,0 +1,239 @@
+"""
+Interface for using the google accessibility ruleset.
+See: https://github.com/GoogleChrome/accessibility-developer-tools
+"""
+import json
+import logging
+import os
+import requests
+
+from collections import namedtuple
+from textwrap import dedent
+
+from .a11y_audit import A11yAudit, A11yAuditConfig, AccessibilityError
+
+log = logging.getLogger(__name__)
+AuditResults = namedtuple('AuditResults', 'errors, warnings')
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class AxsAuditConfig(A11yAuditConfig):
+    """
+    The `AxsAuditConfig` object defines the options available when
+    running an `AxsAudit`.
+    """
+    def __init__(self, *args, **kwargs):
+        super(AxsAuditConfig, self).__init__(*args, **kwargs)
+        self.rules_file = None
+        self.scope, self.rules_to_run, self.rules_to_ignore = None, None, None
+        self.rules_file = os.path.join(
+            os.path.split(CUR_DIR)[0],
+            'vendor/google/axs_testing.js'
+        )
+        self.set_rules([], ignore=False)
+        self.set_rules([], ignore=True)
+        self.set_scope()
+
+    def set_rules(self, rules, ignore=False, include_type="rule"):
+        """
+        Sets `rules_to_run` or `rules_to_ignore` to the passed list of rule IDs.
+
+        Args:
+
+            rules: List of rules to ignore or run for accessibility errors on
+                the page. If `ignore=False`, an empty list means to check for
+                all available rules and `None` means that no audit should be
```

Comment:
Could you please change this docstring to read more clearly? The docstring for set_rules in AxeCoreAuditConfig is great, and really easy to understand.
Maybe for this one break the explanation out into what it means if rules is None, and what it means if rules is an empty list. And in each of those cases explain what it means if ignore=False vs. ignore=True.